client/src/modules/webhooks/Webhooks.tsx

```diff
@@ -0,0 +1,79 @@
+import { useAppByIdQuery } from '../../generated/graphql';
+
+import { Grid, GridItem, Text, Link, Heading } from '@chakra-ui/react';
+import { config } from '../../config';
+import { useState } from 'react';
+
+interface AppDomainProps {
+  appId: string;
+}
+
+export const Webhooks = ({ appId }: AppDomainProps) => {
+  const { data, loading /* error */ } = useAppByIdQuery({
+    variables: {
+      appId,
+    },
+    ssr: false,
+    skip: !appId,
+  });
+  const [isSecretVisible, setIsSecretVisible] = useState(false);
+
+  if (!data) {
+    return null;
+  }
+
+  // // TODO display error
+
+  if (loading) {
+    // TODO nice loading
+    return <p>Loading...</p>;
+  }
+
+  const { app } = data;
+
+  if (!app) {
+    // TODO nice 404
+    return <p>App not found.</p>;
+  }
+
+  return (
+    <>
+      <Heading size="sm" className="text-md font-bold pb-1 py-5">
+        Github webhooks secret
+      </Heading>
+
+      <Text color="gray.400" fontSize="sm">
+        Use this secret when setting up wehbooks for current app
+      </Text>
+      <Grid templateColumns="repeat(5, 1fr)" gap={1}>
+        <GridItem colSpan={2} pt={3}>
+          <Text>
+            {isSecretVisible
+              ? data?.app?.githubRepoId + config.githubClientId.slice(0, 4)
+              : '*************'}
+          </Text>
+        </GridItem>
+        <GridItem colSpan={2} ml={-6}>
+          <svg
```

Comment:
can you remove the className and use chakra values?

you can use `chakra.svg` to use the custom props